virl2_client/models/node.py

```diff
@@ -788,20 +800,32 @@ def tags(self) -> list[str]:
 
         :returns: A list of tags.
         """
-        self.lab.sync_topology_if_outdated()
+        self._lab.sync_topology_if_outdated()
         return self._tags
 
     @locked
-    def add_tag(self, tag: str) -> None:
+    def add_tag(
+        self, tag: str, return_annotation: bool = False
+    ) -> SmartAnnotation | None:
         """
-        Add a tag to this node.
+        Add a tag to this node and return the tag's smart annotation if requested.
 
         :param tag: The tag to add.
+        :param return_annotation: Whether to return the corresponding annotation.
```

Comment:
Personally I do, but I added it mostly because we already fetch the annotation when checking if a sync is necessary, so we can just toss it if you prefer

Reply:
If you really want it gone feel free to apply the suggestions above